docs/developer/local-docker.md

```diff
@@ -50,6 +70,118 @@ This will launch the integration test  runner against the CI environment, using
+- create `test/SOMETESTUSER.tok` (or using the actual username as changed above) and make the contents the token string.
+
+### Other conveniences
+
+The unit tests, by default, are quite noisy. To quiet them down make the following changes to the test configuration:
+
+(all of these changes may not be necessary, but the combination of them does inhibit all logging, with just test results left)
+
+- in `test/unit/karma.conf.js` change `logLevel: config.LOG_INFO,` to `logLevel: config.LOG_DISABLE,`
+
+after the line:
+
+```javascript
+    config.reporters = ['kjhtml', 'brief'];
+```
+
+add this:
+
+``` javascript
+    config.briefReporter = {
+        suppressBrowserLogs: true
+    }
+```
+
+Now your tests, if they pass, will be quite pleasant to gaze upon:
+
+```shell
+erikpearson@Eriks-MacBook-Pro narrative % NARRATIVE_SERVER_URL=http://localhost:8888 npm run test_local
+
+> kbase-narrative-core@5.1.3 test_local /Users/erikpearson/Work/KBase/2022/helpdesk/PTV-1620/pr2redux/narrative
+> karma start test/unit/karma.local.conf.js
+
+
+Chrome Headless 108.0.5359.124 (Mac OS 10.15.7) WARN: 'Loaded token file test/SOMETESTUSER.tok'
+TOTAL: 13 SUCCESS 13 passed      0 failed      0 skipped            
+   13 total       13 passed      0 failed      0 skipped 
+```
+
+To focus tests, you can modify `karma.local.conf.js`. Here we have focussed on the tests that
+are directly contained within `test/unit/spec/widgets/*`, which at the moment, is just `kbaseTabs-spec.js`.
+
+```javascript
+ .concat([
+            // To isolate groups of tests, uncomment ALL of the
+            // exclude filters below, and comment the group you
+            // want to test.
+            //
+            'test/unit/spec/*.js',
+            'test/unit/spec/api/**/*',
+            'test/unit/spec/appWidgets/**/*',
+            'test/unit/spec/common/**/*',
+            'test/unit/spec/function_output/*.js',
+            'test/unit/spec/function_output/kbaseSampleSet/**/*',
+            'test/unit/spec/function_output/modeling/**/*',
+            'test/unit/spec/function_output/rna-seq/**/*',
+            'test/unit/spec/jsonrpc/**/*',
+            'test/unit/spec/narrative_core/**/*',
+            'test/unit/spec/nbextensions/**/*',
+            'test/unit/spec/util/**/*',
+            'test/unit/spec/vis/**/*',
+            'test/unit/spec/widgets/common/*',
+            // 'test/unit/spec/widgets/*'
+        ]);
+```
+
+You may also focus or exclude individual tests using the usual `f*` and `x*` prefixes
+for `describe` and `it`. 
+
+E.g. to only run a test group you can focus the `describe` function by prefixing the it
+with an `f`:
+
+```javascript
+    fdescribe('The KBaseTabs widget', () => {
+      ..
+    }
+```
+
+or to exclude a cranky test you can prefix `it` with `x`:
+
+```javascript
+          xit('should render with minimal options', () => {
+            const kbaseTabs = makeKBaseTabs({
+```
```

Comment:
Great docs. I appreciate that you write documentation as these kinds of things are so easy to forget as soon as you move on to another project.

Reply:
no kidding.